instance/templates/instance/ansible/vars.yml

```diff
@@ -26,19 +31,11 @@ CMS_HOSTNAME: '{{ instance.studio_domain_nginx_regex }}'
 EDXAPP_SESSION_COOKIE_DOMAIN: ''
 
 # Nginx
-{% if instance.protocol == 'https' %}
-NGINX_ENABLE_SSL: true
-NGINX_REDIRECT_TO_HTTPS: true
-{% else %}
+NGINX_SET_X_FORWARDED_HEADERS: false
+
+# SSL is handled on the load balancer, and the appservers are HTTP only.
 NGINX_ENABLE_SSL: false
```

Comment:
@bradenmacdonald At some point in the past, setting this to `true` without providing a certificate resulted in a deployment failure, since the snakeoil cert wasn't available at the path given by default.  I'm not sure whether this has been fixed in the meantime.  I'll test it on stage.